arwen/cryptoapi/cryptoei.go

```diff
@@ -49,6 +49,31 @@ const secp256k1UncompressedPublicKeyLength = 65
 const secp256k1SignatureLength = 64
 const curveNameLength = 4
 
+const (
+	sha256Name                      = "sha256"
+	keccak256Name                   = "keccak256"
+	ripemd160Name                   = "ripemd160"
+	verifyBLSName                   = "verifyBLS"
+	verifyEd25519Name               = "verifyEd25519"
+	verifySecp256k1Name             = "verifySecp256k1"
+	verifyCustomSecp256k1Name       = "verifyCustomSecp256k1"
+	encodeSecp256k1DerSignatureName = "encodeSecp256k1DerSignature"
+	addECName                       = "addEC"
+	doubleECName                    = "doubleEC"
+	isOnCurveECName                 = "isOnCurveEC"
+	scalarBaseMultECName            = "scalarBaseMultEC"
+	scalarMultECName                = "scalarMultEC"
+	marshalECName                   = "marshalEC"
+	unmarshalECName                 = "unmarshalEC"
+	marshalCompressedECName         = "marshalCompressedEC"
+	unmarshalCompressedECName       = "unmarshalCompressedEC"
+	generateKeyECName               = "generateKeyEC"
+	createECName                    = "createEC"
+	getCurveLengthECName            = "getCurveLengthEC"
+	getPrivKeyByteLengthECName      = "getPrivKeyByteLengthEC"
+	ellipticCurveGetValuesName      = "ellipticCurveGetValues"
```

Comment:
As a future improvement, we could use these constants in import builder methods such as `CryptoImports`. Not for this PR.